schema/core.md

```diff
@@ -158,11 +158,13 @@ creators is preserved.
  }
 ```
 
-### Date Created
+### Date Created, Date Modifie, and Date Published
```

Comment:
Fix misspelling of "Modified"

Reply:
Done! The commit 329a0cb resolves this issue.